galley/pkg/runtime/state.go

```diff
@@ -319,3 +321,12 @@ func (s *State) String() string {
 
 	return b.String()
 }
+
+func monitorEntry(resourceKey resource.VersionedKey, added bool) {
+	namespace, name := resourceKey.FullName.InterpretAsNamespaceAndName()
+	count := 1
+	if !added {
+		count = -1
```

Comment:
Do we want the value to be `0` for a resource stat?  Or should we just delete the metric (with the particular labeling) altogether? I feel like it might be cleaner to just delete. It would probably make querying, etc., easier too. Thoughts?

Reply:
Didn't find a way to delete it... https://godoc.org/go.opencensus.io/stats#Record?

Reply:
Interesting. OK. We should open an issue with the opencensus team to start a dialog. In the meantime, we'll have to get smarter about queries.